CODIGO/FrmLogear.frm

```diff
@@ -392,21 +392,16 @@ Private Sub cmdIngresar_Click()
         #End If
         
         #If PYMMO = 1 Then
+            'DEVELOPER mode is used to connect to localhost
             #If DEVELOPER = 1 Then
                 IPdelServidorLogin = "127.0.0.1"
                 PuertoDelServidorLogin = 4000
                 IPdelServidor = IPdelServidorLogin
                 PuertoDelServidor = 7667
             #Else
-                'Production and staging use this path
-                #If DEBUGGING = 0 Then
-                    Call SetDefaultServer
-                #Else
-                    IPdelServidorLogin = "45.235.98.31"
-                    PuertoDelServidorLogin = 11814
-                #End If
+                'When not in DEVELOPER mode we read the ip and port from the list
+                Call SetDefaultServer
```

Comment:
why did you remove staging setting here?

Reply:
Because it's confusing that when I use DEBUG = 1 the client would not connect to prod. 

We need a new define `CONNECT_TO_STAGING` , `DEBUG` should not affect where the client connects to.